test/patterns/private_static_method/test_private_static_method.py

```diff
@@ -0,0 +1,13 @@
+
+import os
+from unittest import TestCase
+from aibolit.patterns.private_static_method.private_static_method import PrivateStaticMethod
+from pathlib import Path
+
+
+class TestPrivateStaticMethod(TestCase):
+    dir_path = Path(os.path.realpath(__file__)).parent
+
+    def test_find_non_final_atributes(self):
+        lines = PrivateStaticMethod().value(Path(self.dir_path, 'PrivateStaticMethod.java'))
+        self.assertEqual(1, len(lines))
```

Comment:
Isn't it better to check for line number explicitly? `assertEqual(lines, [2])`

Reply:
@tribals We don't have this information on the tree generated by AST